redash/schedule.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+from datetime import datetime, timedelta
+from functools import partial
+from random import randint
+
+from rq_scheduler import Scheduler
+
+from redash import settings, redis_connection
+from redash.tasks import (sync_user_details, refresh_queries,
+                          empty_schedules, refresh_schemas,
+                          cleanup_query_results,
+                          version_check, send_aggregated_errors)
+
+rq_scheduler = Scheduler(connection=redis_connection,
```

Comment:
In Python 3 we use `decode_responses=True`, while `rq-scheduler` expects it to be `False` and uses `decode` on values it receives:

https://github.com/rq/rq-scheduler/blob/061488e79b82af215f7f98fa1554a0aa84dfd62f/rq_scheduler/scheduler.py#L321

When porting this to Python 3, we should probably create a Redis connection for it without the `decode_responses` change :thinking: We can create a "factory" function for creating the Redis connection and move the logic of mangling the Redis URL there (and make it optional).

Another option to consider is to revert to not decoding responses and do it where needed. It just felt redundant, so I was hoping to avoid it.

@rauchy @NicolasLM